radis/test/tools/test_database.py

```diff
@@ -115,17 +115,43 @@ def test_plot_spec(plot=True, close_plots=True, verbose=True, *args, **kwargs):
         plot_spec(getTestFile("N2C_specair_380nm.spec"))
 
 
-def _run_testcases(plot=True, close_plots=False, verbose=True, *args, **kwargs):
-
-    test_speclist(*args, **kwargs)
-    test_database_functions(
-        plot=plot, close_plots=close_plots, verbose=verbose, *args, **kwargs
+def test_save_compressed2():
+    "Check if saving a spectrum with compress = 2 does not change something."
+    from radis import calc_spectrum, SpecDatabase
+    import shutil
+
+    # s1 : cm-1, vacuum
+    # s2 : nm, air
+    s = calc_spectrum(
+        1900,
+        1950,  # cm-1
+        # wavelength_min=4160,   # nm
+        # wavelength_max=4190,   # nm
+        molecule="CO",
+        isotope="1,2,3",
+        pressure=1.01325,  # bar
+        Tgas=700,  # K
+        mole_fraction=0.1,
+        path_length=1,  # cm
+        verbose=False,
+        wstep=0.01,
+        medium="vacuum",
     )
-    test_plot_spec(plot=plot, close_plots=close_plots, verbose=verbose, *args, **kwargs)
 
-    return True
+    db = SpecDatabase(dirname(getTestFile(".")) + "/newDb/")
+    db.add(s, compress=1, if_exists_then="error")
+
+    # s_bis = db.get_closest(Tgas=700)
+    s_bis = db.get_unique(Tgas=700)
```

Comment:
I disagree: here we're not reading it from the disk, but directly from the SpecDatabase. In the past spectra were stored on disk & reloaded before they were added in the database, but since #84 we store them directly, no? 

So you can:
- generate a new Database db2. Same place. 
- or simply use `s.store()` and `s=load_spec(...)`. Do we really need the SpecDatabase there?